reproducibility_project/src/engines/lammps-VU/project.py

```diff
@@ -0,0 +1,237 @@
+    )  # save to gromacs topology for later conversions in mdtraj
+    write_lammpsdata(
+        typed_surface,
+        "box.lammps",
+        atom_style="full",
+        unit_style="real",
+        mins=[system.get_boundingbox().vectors[0]],
+        maxs=[system.get_boundingbox().vectors[1]],
+        use_rb_torsions=True,
+    )  # write out a lammps topology
+    return
+
+
+@Project.operation
+@Project.pre(lambda j: j.sp.engine == "lammps-VU")
+@Project.pre(lammps_created_box)
+@Project.post(lammps_copy_files)
+@flow.with_job
+@flow.cmd
+def lammps_cp_files(job):
+    """Copy over run files for lammps and the PBS scheduler."""
+    lmps_submit_path = "../../src/engine_input/lammps/VU_scripts/submit.pbs"
+    lmps_run_path = "../../src/engine_input/lammps/input_scripts/in.*"
+    msg = f"cp {lmps_submit_path} {lmps_run_path} ./"
+    return msg
+
+
+@Project.operation
+@Project.pre(lambda j: j.sp.engine == "lammps-VU")
+@Project.pre(lammps_copy_files)
+@Project.post(lammps_minimized_equilibrated_nvt)
+@flow.with_job
+@flow.cmd
+def lammps_em_nvt(job):
+    """Run energy minimization and nvt ensemble."""
+    in_script_name = "in.minimize"
+    r_cut = job.sp.r_cut * 10
+    modify_submit_scripts(in_script_name, job.id)
+    msg = f"qsub -v 'infile={in_script_name}, seed={job.sp.replica+1}, T={job.sp.temperature}, P={job.sp.pressure}, rcut={r_cut}' submit.pbs"
+    return msg
+
+
+@Project.operation
+@Project.pre(lambda j: j.sp.engine == "lammps-VU")
+@Project.pre(lammps_minimized_equilibrated_nvt)
+@Project.post(lammps_equilibrated_npt)
+@flow.with_job
+@flow.cmd
+def lammps_equil_npt(job):
+    """Run npt ensemble equilibration."""
+    in_script_name = "in.equilibration"
+    modify_submit_scripts(in_script_name, job.id)
+    r_cut = job.sp.r_cut * 10
+    msg = f"qsub -v 'infile={in_script_name}, seed={job.sp.replica+1}, T={job.sp.temperature}, P={job.sp.pressure}, rcut={r_cut}' submit.pbs"
+    return msg
+
+
+@Project.operation
+@Project.pre(lambda j: j.sp.engine == "lammps-VU")
+@Project.pre(lammps_equilibrated_npt)
+@Project.post(lammps_production)
+@flow.with_job
+@flow.cmd
+def lammps_prod(job):
+    """Run npt ensemble production."""
+    in_script_name = "in.production"
+    modify_submit_scripts(in_script_name, job.id)
+    r_cut = job.sp.r_cut * 10
+    msg = f"qsub -v 'infile={in_script_name}, seed={job.sp.replica+1}, T={job.sp.temperature}, P={job.sp.pressure}, rcut={r_cut}' submit.pbs"
+    return msg
+
+
+@Project.operation
+@Project.pre(lambda j: j.sp.engine == "lammps-VU")
+@Project.pre(lammps_production)
+@Project.post(lambda j: j.isfile("prod.gsd"))
+@flow.with_job
+@flow.cmd
+def lammps_calc_rdf(job):
+    """Create an rdf from the gsd file using Freud analysis scripts."""
+    # Create rdf data from the production run
+    import mbuild as mb
+    import mdtraj
+
+    traj = md.load("prod.xtc", top="box.gro")
+    traj.save("prod.gsd")
+    # TODO: Use freud rdf PR to create an RDF from the gsd file
+    return
```

Comment:
Just a heads up, this should not be handled here in this project.py. That will be handled solely by the `project-analysis.py` file located with the `init.py, src/ ` etc.

So instead you should just need a method to convert your prod traj to `trajectory.gsd`

Reply:
Noted! We should be good to go then.